core/tauri/src/lib.rs

```diff
@@ -625,6 +625,7 @@ pub trait Manager<R: Runtime>: sealed::ManagerBase<R> {
   ///   app.emit_all("synchronized", ());
   /// }
   /// ```
+  #[tracing::instrument("app::emit::all", skip(self, payload))]
```

Comment:
Why are we skipping payload? shouldn't the payload be instrumented as well? to differentiate which emit call are we tracking?

Reply:
the payload is added on a child span though

Reply:
I guess we should add a TODO then for v2